site/docs/components/combo-box/examples.mdx

```diff
@@ -0,0 +1,68 @@
+---
+# Leave the frontmatter as is
+title:
+  $ref: ./#/title
+layout: DetailComponent
+sidebar:
+  exclude: true
+data:
+  $ref: ./#/data
+---
+
+<LivePreviewControls>
+<LivePreview componentName="combo-box" exampleName="Default">
+
+### Default (single select)
+
+A single item can be selected from the combo box options.
+
+Combo Box contains an input field that opens a list. As users' type, the list displays all filtered options based on the user’s input.
+
+</LivePreview>
+
+<LivePreview componentName="combo-box" exampleName="CustomRenderer" >
+
+### Custom renderer
+
+Customize the list item display to visually assist users, for example rendering the flag icons next to the name of countries as seen in this example. Use the `ListItem` prop to customize list items.
+
+Guidance:
```

Comment:
```suggestion
#### Guidance
```

Reply:
keeping this in line with other docs from the same batch, as per design's guidance